plugins/ruby-foreman-azure/debian/gem.list

```diff
@@ -0,0 +1,2 @@
+# list of gem urls to download via Jenkins, space separated
+GEMS="https://rubygems.org/downloads/foreman_azure-1.0.0.gem"
```

Comment:
This will also need every dependency of foreman_azure - i.e. fog-azure, azure (and recurse!)